lambdas/data-migration2/tests/test-granulesAndFiles.js

```diff
@@ -504,29 +509,37 @@ test.serial('migrateFileRecord handles nullable fields on source file data', asy
   );
 });
 
-test.serial('migrateGranulesAndFiles skips already migrated granule record', async (t) => {
+test.serial('migrateGranuleAndFilesViaTransaction skips already migrated granule record', async (t) => {
   const {
     knex,
     testGranule,
   } = t.context;
 
-  await migrateGranuleRecord(testGranule, knex);
-  await granulesModel.create(testGranule);
+  await migrateGranuleAndFilesViaTransaction(
+    testGranule,
+    {},
+    knex,
+    1
+  );
 
   t.teardown(() => {
     granulesModel.delete(testGranule);
   });
 
-  const migrationSummary = await migrateGranulesAndFiles(process.env, knex);
-  t.deepEqual(migrationSummary, {
+  const result = await migrateGranuleAndFilesViaTransaction(
+    testGranule,
+    {},
+    knex,
+    1
+  );
+  t.deepEqual(result, {
     filesResult: {
       total_dynamo_db_records: 1,
       failed: 0,
-      skipped: 0,
+      skipped: 1,
```

Comment:
Good catch (this and the previous commit).

Reply:
I do wonder more generally if we should make this report object more clear in that it means granule files and not the files reference objects, but we shouldn't address that in this PR, if at all.